### PR TITLE
fix some smal things

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue # .

### CODE REVIEW

File: src/administrator/components/com_weblinks/helpers/html/weblink.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 /**
  * Weblink HTML helper class.
  *
- * @since  1.6
+ * @since  __DELPOY_VERSION__
```
Comment (infograf768) [inline src/administrator/components/com_weblinks/helpers/html/weblink.php:19]: DEPLOY I guess :)

Comment (zero-24) [reply src/administrator/components/com_weblinks/helpers/html/weblink.php:19]: opps can you fix it?

Comment (zero-24) [reply src/administrator/components/com_weblinks/helpers/html/weblink.php:19]: or shoud i do a seccond PR?

Comment (infograf768) [reply src/administrator/components/com_weblinks/helpers/html/weblink.php:19]: done

Comment (zero-24) [reply src/administrator/components/com_weblinks/helpers/html/weblink.php:19]: :+1:

**Pull request merged**